src/main/java/net/starlark/java/eval/MethodDescriptor.java

```diff
@@ -73,6 +85,21 @@ private MethodDescriptor(
     this.allowReturnNones = allowReturnNones;
     this.useStarlarkThread = useStarlarkThread;
     this.useStarlarkSemantics = useStarlarkSemantics;
+
+    if (method.getReturnType() == void.class) {
```

Comment:
For brevity, declare a local ret = getReturnType().
Order the comparisons in the natural way: if (variable == constant)